docs/testplan/multi_tier_capacity_test.md

```diff
@@ -0,0 +1,70 @@
+# SONiC Switch Capacity Test
+
+## Test Objective
+
+This test aims to assess the true capacities of SONiC switches.
+
+## Test Setup
+
+The test is designed to be topology-agnostic, meaning it does not assume or impose a specific setup wiring, settings, or configuration. The running script will track power usage, temperature, queue watermark, and health state of all SONiC switches in a testbed.
+
+## Metrics to Check
+
+- **PSU power usage**: `show platform psu`.
+- **Sensor temperature**: `show platform temperature`. Among the outputs, “Cpu temp sensor” and “Switch Card temp sensor” are of interest.
+- **Queue watermark**: `show queue watermark`.
+- **Interface counters**: `show interface counters`. (The outputs include drop counters.)
+
+## Test Steps
```

Comment:
Move test steps up before metrics to check. and merge the metrics to check with the last section.

Each metrics can be its own sub sections.